src/lib.rs

```diff
@@ -1,3 +1,8 @@
+//! This crate provides a `Watch` that launch a given command, re-launching this
+//! command when changes are detected in your source code.
```

Comment:
```suggestion
//! A [clap](link-to-clap) "watch" command for your [xtask](link-to-xtask).
//!
//! When the command is ran it:
//!  -  starts the provided command in background
//!  -  detect changes in the path provided
//!  -  re-start the command in background if changes are detected
//!
//! # Example
//!
//! ```rust
//! <copy example from demo>
//! ``
```

Reply:
I will use this changes in another PR that will be focus on the documentation